autogen/main.tf.tmpl

```diff
@@ -159,7 +176,11 @@ resource "google_compute_backend_service" "default" {
     }
   }
 
-  depends_on = [google_compute_health_check.default]
+  depends_on = [
+    {% if not serverless %}
+    google_compute_health_check.default
```

Comment:
Shouldn't the conditional cover the whole depends_on to prevent it from being empty?
https://github.com/terraform-google-modules/terraform-google-lb-http/blob/249f34825ce3f6c626206783d2b6d4f6e1eb69c1/modules/serverless_negs/main.tf#L151

Reply:
Yeah that would be slightly cleaner but NBD either way.

Reply:
Ack. Wasn't sure. I left it there for future extensibility. But removed now.